tests/fetch/example_synchronous_fetch.cpp

```diff
@@ -16,7 +17,6 @@ int main()
   attr.attributes = EMSCRIPTEN_FETCH_LOAD_TO_MEMORY | EMSCRIPTEN_FETCH_SYNCHRONOUS;
   emscripten_fetch_t *fetch = emscripten_fetch(&attr, "gears.png");
   printf("Fetch finished with status %d\n", fetch->status);
-#ifdef REPORT_RESULT
-  REPORT_RESULT(fetch->status);
-#endif
+  assert(fetch->status == 200);
```

Comment:
Is the assertion failure the expected way to fail? Other tests use `return 1`, so...

Reply:
Assertion failure causes the tests to abort which generates a non-zero return code yes.